<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab2</class>
 <widget class="QWidget" name="Lab2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOpen">
         <property name="text">
          <string>Открыть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Кривая по 4-ём точкам</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineSecondX">
            <property name="text">
             <string>15</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineThirdX">
            <property name="text">
             <string>210</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="lineFourX">
            <property name="text">
             <string>312</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineFirstX">
            <property name="text">
             <string>101</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineSecondY">
            <property name="text">
             <string>111</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineThirdY">
            <property name="text">
             <string>30</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineFourY">
            <property name="text">
             <string>72</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineFirstY">
            <property name="text">
             <string>13</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Нарисовать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Сглаживание ломаной линии</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QLineEdit" name="pointCount">
            <property name="text">
             <string>12</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" colspan="2">
           <widget class="QPushButton" name="antialiasingButton">
            <property name="text">
             <string>Сгладить</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QPushButton" name="polylineButton">
            <property name="text">
             <string>Ломанная</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Количество точек:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineFirstX</tabstop>
  <tabstop>lineFirstY</tabstop>
  <tabstop>lineSecondX</tabstop>
  <tabstop>lineSecondY</tabstop>
  <tabstop>lineThirdX</tabstop>
  <tabstop>lineThirdY</tabstop>
  <tabstop>lineFourX</tabstop>
  <tabstop>lineFourY</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pointCount</tabstop>
  <tabstop>polylineButton</tabstop>
  <tabstop>antialiasingButton</tabstop>
  <tabstop>buttonOpen</tabstop>
  <tabstop>buttonSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
